src/cascadia/TerminalApp/App.cpp

```diff
@@ -1216,12 +1217,12 @@ namespace winrt::TerminalApp::implementation
 
         if (tabIndexFromControl == focusedTabIndex)
         {
-            if (focusedTabIndex >= _tabs.size())
+            auto const focusedTabCount = gsl::narrow_cast<decltype(focusedTabIndex)>(_tabs.size());

```

Comment:
this is still just a `tabCount`, doesn't need to have `focused` in the name :smile:

Reply:
Duh... 
Thanks.